packages/team-management/src/components/TeamsAddEdit/Form.tsx

```diff
@@ -194,12 +194,7 @@ export const Form: React.FC<Props> = (props: Props) => {
         label={lang.TEAM_MEMBERS}
         tooltip={lang.TIP_REQUIRED_FIELD}
       >
-        <Select
-          allowClear
-          mode="multiple"
-          placeholder={lang.SELECT_PRACTITIONER}
-          value={initialValue.practitionersList.map((practitioner) => practitioner.name)}
-        >
+        <Select allowClear mode="multiple" placeholder={lang.SELECT_PRACTITIONER}>
```

Comment:
does initial value works properly after removal of value prop ?

Reply:
Yes, but with identifier. The name is picked from the options provided to Option